lib/Service/TagService.php

```diff
@@ -82,11 +83,11 @@ public function setTag(int $fileId, string $tagName): void {
 
 		if (isset($filesTagIds[$fileId])) {
 			foreach ($filesTagIds[$fileId] as $tagId) {
-				if ($tagId != $tag->getId() && \in_array($tagId, $vaasTagIds)) {
```

Comment:
has this change any value?

The slash is making it clear to use the function from the [global namespace](https://www.php.net/manual/en/language.namespaces.global.php).